### DISCUSSION TITLE
Forgot Self-control app password. Need to reset. I am dependent on this app. to get any work done.

### DISCUSSION THREAD

**Discussion by Kscoop** [SelfControlApp/selfcontrol#765] [Q&A]
Please help me figure out how to reset my self-control password. I've tried reloading the app but it doesn't work. I'm not very internet savvy. I use this app daily to help me get my work done without being distracted by internet. I forgot my password while on break and cannot figure out how to set a new one. Please email me at knoxscooptoo@gmail.com

**Comment by slambert** [comment]
There's not a unique password for the program. The password is your login password for OSX.
